base/src/main/java/com/smartdevicelink/proxy/rpc/VideoStreamingCapability.java

```diff
@@ -180,4 +184,14 @@ public VideoStreamingCapability setScale(Double scale) {
         setValue(KEY_SCALE, scale);
         return this;
     }
+
```

Comment:
@kostyaBoss above line 188 you can add these lines (code generator tool):

"    /**
     * Gets the additionalVideoStreamingCapabilities.
     *
     * @return List<VideoStreamingCapability>
     * {"array_min_size": 1, "array_max_size": 100}
     * @since SmartDeviceLink 7.0.0
     */"